test-harness/magefile.go

```diff
@@ -93,7 +93,7 @@ func verifyRunsQuietly(cmd string, args ...string) error {
 		return nil
 	}
 
-	return fmt.Errorf("ERROR: command '%s' with arguments %s failed. Output was: '%s'", cmd, args, output)
+	return fmt.Errorf("ERROR: command '%s' with arguments %s failed. Output was: '%s'. %s", cmd, args, output, instructionsToFix)
```

Comment:
I like this. Looks like a very complete message that should lead users to understand what went wrong.